server/db/migrations/19_create_host_node_stats_created_at_index.rb

```diff
@@ -0,0 +1,5 @@
+class CreateHostNodeStatsCreatedAtIndex < Mongodb::Migration
+  def self.up
+    HostNodeStat.create_indexes
```

Comment:
Kind of... I added the created_at index to the model definition for #1908, but it wasn't necessary at that point as we weren't using it yet.  I still need to verify if that index is enough for the aggregations being done in this PR, or if it needs to be modified (ie making it a compound index).